File: 8KYU/Contamination #1 String/solution.cs
using System.Linq;
public class Kata
{
  public static string Contamination(string text, string character)
     => (string.IsNullOrEmpty(text) || string.IsNullOrEmpty(character))
           ? string.Empty
           : string.Join("", Enumerable.Repeat(character, text.Length));
}

File: 8KYU/Contamination #1 String/solution.rs
fn contamination(text: &str, character: &str) -> String {
    if text == "" || character == "" {
        return "".to_string();
    }
    let mut s: String = "".to_string();
    
    for _ in 0..text.chars().count() {
        s = s + character;
    }
    return s;
}
